### PR TITLE
added optimisting rendered vote button and tested for error handling

### PR DESCRIPTION
**Pull request by alexmbentley**

### CODE REVIEW

**Comment by pkaodev** [comment]
Good start, the votes are being sent to the database and permanently staying the same on refresh.

However, I can click the upvote button as many times as I want.  Let's change it so that once I have pressed it, I can't any more.  It is ok if I can refresh and press again, but I shouldn't be able to do it without reloading the page.

**Comment by pkaodev** [comment]
Send in another PR after this has been changed

File: src/components/SingleReview.jsx [hunk 1]
```diff
@@ -24,6 +24,22 @@ const SingleReview = () => {
         }
       });
   }, [id]);
+
+  useEffect(() => {
+    getComments(id)
+      .then(({ data }) => {
+        console.log(data, '<<< comments data');
```
Comment (hevmag) [inline src/components/SingleReview.jsx:31]: remove console.log before merge

File: src/components/SingleReviewCard.jsx [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+        setVotes(0);
+        setButtonStatus(false);
+        setIsError('Something went wrong');
+      });
+    }
+  };
+
   return (
     <div className="reviewCard">
       {console.log(review, '<<Review in here')}
```
Comment (hevmag) [inline src/components/SingleReviewCard.jsx:26]: remove console.log here too

**Review by hevmag — APPROVED** [review]
looks good to merge! just make sure to remove those console.logs and make sure we aren't getting anything in the console before merging

**Pull request merged**